### PR TITLE
[jsk_2016_baxter_apc] remember angle-vector to pull out arm from bin

### PR DESCRIPTION
**Pull request by knorth55**
this is for ik-unsolvable region
#1209

### CODE REVIEW

File: jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l [hunk 1]
```diff
@@ -394,20 +395,19 @@
         )
       (when do-stop-grasp (unless graspingp (send self :stop-grasp arm)))
       (send self :angle-vector-sequence
-            (list
-              (send *baxter* arm :move-end-pos #f(0 0 50) :world)
-              (send self :ik->bin-entrance arm bin)
-              )
-             :fast nil 0 :scale 5.0)
+         (reverse avs)
+         ;;(list
```
Comment (k-okada) [inline jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l]: please write comment why you did not remove this line, keep code as commented-out

File: jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l [hunk 2]
```diff
@@ -394,20 +395,19 @@
...
       (send self :wait-interpolation)
       ;; move arm in-bin -> bin-entrance
       (send self :angle-vector-sequence
         (list
           (send *baxter* arm :move-end-pos #f(0 0 50))
           (send self :ik->bin-entrance arm bin :offset #f(0 0 50))
           (send self :ik->bin-entrance arm bin :offset #f(-100 0 50))
-          (send self :ik->bin-entrance arm bin :offset #f(-300 0 50))
-          )
+          (send self :ik->bin-entrance arm bin :offset #f(-300 0 50)))
```
Comment (k-okada) [inline jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l]: this is just a style change

**Comment by pazeshun** [comment]
Umm...
This may conflict to my PR.
Please explain why you changed baxter-interface in jsk_2016_01_baxter _apc.

**Comment by knorth55** [comment]
what do you mean?
you can rebase it, can't you?

**Comment by pazeshun** [comment]
I thought @wkentaro 's comment in #1322 suggested new baxter-interface using old robot model in jsk_2015_05_baxter_apc, so I was surprised about your change. 
It's ok to rebase, but conflicts between your PR and mine will often occur.

**Comment by knorth55** [comment]
?

**Comment by pazeshun** [comment]
Ok, I understood. Sorry for confusing.

**Comment by knorth55** [comment]
#1209

**Pull request merged**